tests/test_symbolic_printer_visitor.py

```diff
@@ -85,47 +87,47 @@ def test_real_visitor_hard(self):
                         RealConstant('c1')),
                     RealVariable('y')),
                 RealConstant('c2')))
-        self.assertEqual(tree.accept(SymbolicPrinterVisitor()), "(= z (* (- (+ (/ x c0) c1) y) c2))")
+        symbolic_repr = tree.accept(SymbolicPrinterVisitor())
+        self.assertEqual("(= z (* (- (+ (/ x c0) c1) y) c2))", symbolic_repr)
+
+    def test_string_visitor_hard(self):
+        tree = StringEquality(
+            StringVariable('z'),
+            StringConcatenation(
+                StringReplacement(
+                    StringVariable('x'),
+                    StringLiteral('c0'),
+                    StringLiteral('c1')),
+                Substring(
+                    StringVariable('y'),
+                    StringLength(StringLiteral('c2')),
+                    IntegerConstant('c3'))))
```

Comment:
As in the previous comment, either way are fine, but I would stick with one indentation style for all the tests.